examples/publisher.c

```diff
@@ -125,7 +125,8 @@ static int AddTopics(char* topicList, char** msg, int* keep, int* ttl, int* encr
 
 static void OnAck(DPS_Publication* pub, uint8_t* data, size_t len)
 {
-    DPS_PRINT("Ack for pub UUID %s(%d)\n", DPS_UUIDToString(DPS_PublicationGetUUID(pub)), DPS_PublicationGetSequenceNum(pub));
+    DPS_PRINT("Ack for pub UUID %s(%d) [%s]\n", DPS_UUIDToString(DPS_PublicationGetUUID(pub)),
+              DPS_PublicationGetSequenceNum(pub), KeyIdToString(DPS_AckGetId(pub)));
```

Comment:
not related to this change but what is the difference between the UUID string and the KeyID string?

Reply:
Key ID is just a byte array: it could be a UUID or it could be something else.